commit 6e447fd101557bdc5b72fbbebf1d7da9c6526d8b
Author: John Spray <jspray@redhat.com>
Date:   Wed Apr 11 08:45:53 2018 -0400

    mgr: clean up ActivePyModules logging
    
    This was running the prefix and message together.
    
    Signed-off-by: John Spray <john.spray@redhat.com>

diff --git a/src/mgr/ActivePyModules.cc b/src/mgr/ActivePyModules.cc
index ff4bd2d3ce0..e61030ab686 100644
--- a/src/mgr/ActivePyModules.cc
+++ b/src/mgr/ActivePyModules.cc
@@ -431,7 +431,7 @@ bool ActivePyModules::get_store(const std::string &module_name,
   const std::string global_key = PyModule::config_prefix
     + module_name + "/" + key;
 
-  dout(4) << __func__ << "key: " << global_key << dendl;
+  dout(4) << __func__ << " key: " << global_key << dendl;
 
   if (store_cache.count(global_key)) {
     *val = store_cache.at(global_key);
@@ -453,7 +453,6 @@ bool ActivePyModules::get_config(const std::string &module_name,
 
   dout(4) << __func__ << " key: " << global_key << dendl;
 
-  
   Mutex::Locker lock(module_config.lock);
   
   if (module_config.config.count(global_key)) {
